downloadphp.ps1

```diff
@@ -1,5 +1,31 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-Invoke-WebRequest -Uri https://windows.php.net/downloads/releases/php-7.2.7-nts-Win32-VC15-x64.zip -OutFile php.zip
-Expand-Archive -LiteralPath php.zip -DestinationPath php\
+$workingdir = (Get-Location).Path
+if($PSVersiontable.PSVersion.Major -lt 3)
+{
+    Write-Warning "Please download php and place it in the same folder as this script."
+    Write-Output "Download from: https://windows.php.net/downloads/releases/php-7.2.7-nts-Win32-VC15-x64.zip"
+    Write-Output ("Save to this directory: $workingdir `nand rename it to php.zip" -f (Get-Location).Path)
```

Comment:
`nand?

Reply:
`n is line break character in powershell

Reply:
Well powershell just had to reinvent everything, didn't they